common/src/main/java/com/microsoft/identity/common/nativeauth/internal/util/CommandUtil.java

```diff
@@ -142,4 +143,54 @@ public static SignInSubmitPasswordCommandParameters createSignInSubmitPasswordCo
 
         return commandParameters;
     }
+
+    /**
+     * Adds scopes to [MFASubmitChallengeCommandParameters] object and returns a new
+     * [MFASubmitChallengeCommandParameters] object.
+     * @param parameters input command parameter
+     * @param defaultScopes scopes to be added
+     * @return [MFASubmitChallengeCommandParameters] object with scopes
+     */
+    public static MFASubmitChallengeCommandParameters createMFASubmitChallengeCommandParametersWithScopes(
+            MFASubmitChallengeCommandParameters parameters,
+            List<String> defaultScopes
+    ) {
+        return parameters.toBuilder()
+                .scopes(defaultScopes)
+                .correlationId(parameters.getCorrelationId())
+                .build();
+    }
+
+    /**
+     * Adds continuation token to [SignInStartCommandParameters] object and returns a new
+     * [SignInSubmitPasswordCommandParameters] object.
```

Comment:
nit: Should this "SignInSubmitCodeCommandParameters" ?

Reply:
No, actually this entire comment is wrong.